aerofiles/seeyou/reader.py

```diff
@@ -193,3 +193,90 @@ def decode_description(self, description):
             return None
 
         return description
+
+    def decode_taskname(self, fields):
+        if not fields:
+            raise ParserError('Name field must not be empty')
+
+        return fields[0]
+
+    def decode_tasklist(self, fields, waypoints):
```

Comment:
yes, that would probably be a good idea, but I'd suggest that you focus on the `Reader` for now and once that works properly (feel free to open a work-in-progress PR for your branch) we'll have another look at the `Converter`